packml_ros/src/packml_ros.cpp

```diff
@@ -171,14 +144,32 @@ bool PackmlRos::transRequest(packml_msgs::Transition::Request& req, packml_msgs:
   }
 }
 
+bool PackmlRos::eventRequest(packml_msgs::SendEvent::Request& req, packml_msgs::SendEvent::Response& res)
+{
+  auto event_id = req.event_id;
+  res.result = eventGuard(event_id);
+  return res.result;
+}
+
+bool PackmlRos::triggerStateChange(packml_msgs::InvokeStateChange::Request& req, packml_msgs::InvokeStateChange::Response& res)
+{
+  sm_->invokeStateChangedEvent(req.name, static_cast<packml_sm::StatesEnum>(req.state_enum));
+  return true;
```

Comment:
Should this wrapped in a try/catch?  Otherwise the return value doesn't seem very useful (though I realize that service callbacks have to return a value, so maybe it isn't necessary for it to be meaningful).

Reply:
The return value is mostly an assurance that the call was completed and processed (as opposed to not being available to the client side). There's really no assurances otherwise with the invokeStateChangedEvent, so it doesn't make sense to have a return type like with the other messages.

Reply:
Got it, makes sense.